site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+
+
+*What issues arise with the ability to morally object in the workplace?*
+
+
+However, raising the alarm bell might be more difficult if you are working in a country you haven’t worked in before. It can be less clear what the norms are, and how they should be appropriately challenged. A lot of researchers move internationally for their work, so this might be something that crops up a lot in these communities.
+
+In addition to knowing the norms in place, the size of an organisation might impact your ability to challenge something you morally disagree with. In large organisations, it is much easier to get lost amongst the weeds. Even if you raise your head above the parapet, it could be soon forgotten, increasing the importance of having colleague support.
+
+Difficulties with garnering the support of your colleagues arise with the growth of remote working. Often, these sorts of conversations are important to feel out informally first, to garner perspective and general thoughts. Opportunities for informal interactions with colleagues are heavily reduced in remote work scenarios.
+
+
+*Different levels of authority may have different motivations; how does this affect our freedom to choose ethical paths?*
+
+
+On a personal level, PhDoers might have autonomy to decide not to go down certain paths, but this could be limited by funding. Sometimes there are incentives behind funding bodies that limit the freedom of choice for the direction you take, and it isn’t always possible to know what those incentives really are, or the reasoning behind them. The motivation of other bodies supporting, and to some extent constraining, your work might be different to yours and this might be hidden from you.
+
+Diversion of motivation from authoritative bodies and choice constraint continues in more focussed positions like postdoc projects and industry jobs. It can be pretty cut and dry in these domains: you either do the goals you’re given, or you lose your job! There is also the sunk cost fallacy in defence projects, removing the excuse of “this doesn’t work”. For example, the SA-80 British made assault rifle which is [famously rubbish](https://nationalinterest.org/blog/buzz/introducing-sa80-worst-military-rifle-ever-44987) , and yet the military refused to abandon it.
```

Comment:
Really good article you linked here, there's an absolute gem from the start of it that I thought would be funny to include! But it might break the flow a bit so feel free to disregard
```suggestion
Diversion of motivation from authoritative bodies and choice constraint continues in more focussed positions like postdoc projects and industry jobs. It can be pretty cut and dry in these domains: you either do the goals you’re given, or you lose your job! There is also the sunk cost fallacy in defence projects, removing the excuse of “this doesn’t work”. For example, the SA-80 British made assault rifle which is [famously rubbish](https://nationalinterest.org/blog/buzz/introducing-sa80-worst-military-rifle-ever-44987) and described by some as "the weaponized version of civil servant, as it doesn’t work, and can’t be fired", and yet the British military refused to abandon it. 
```